src/bot.ts

```diff
@@ -76,33 +76,36 @@ export class DiscordBot {
     private userSync: UserSyncroniser;
     private channelSync: ChannelSyncroniser;
     private roomHandler: MatrixRoomHandler;
+    private provisioner: Provisioner;
+    private botUserId: string;
 
     /* Handles messages queued up to be sent to discord. */
     private discordMessageQueue: { [channelId: string]: Promise<void> };
 
-    constructor(config: DiscordBridgeConfig, store: DiscordStore, private provisioner: Provisioner) {
+    constructor(botUserId: string, config: DiscordBridgeConfig, bridge: Bridge) {
```

Comment:
```suggestion
    constructor(private botUserId: string, config: DiscordBridgeConfig, private bridge: Bridge) {
```

Reply:
uuuuh, they are set right after to the private vars?

Currently there is a mixture when setting private vars passed to the constructor which way it should be done, so, if soru changes this she should go over the other ones and change it there, too.

Sooo... TL;DR which syntax should we use? the one you suggested?

Reply:
I kiiiinda prefer the syntax suggested. The reason it wasn't done is because I don't believe it was supported when we started the project (or I didn't notice, one of them). I very much prefer saving us two lines :)